packages/config/src/projects/layer2s/scroll.ts

```diff
@@ -407,9 +412,13 @@ export const scroll: Layer2 = {
       ...FORCE_TRANSACTIONS.SEQUENCER_NO_MECHANISM,
       references: [
         {
-          text: 'EnforcedTxGateway.sol - Etherscan source code, EnforcedTxGateway is paused',
+          text: 'EnforcedTxGateway.sol - Etherscan source code',
           href: 'https://etherscan.io/address/0x642af405bF64660665B37977449C9C536B806318#code',
         },
+        {
+          text: 'EnforcedTxGateway is paused - Etherscan proxy contract',
```

Comment:
this is for clarity, because when opening `https://etherscan.io/address/0x642af405bF64660665B37977449C9C536B806318#code` to check the "paused" value, it shows "false" because the states are all stored in the proxy contract. there are dummy values in the implementation contract.